<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKToolBoxUiInputMenu</class>
 <widget class="QWidget" name="SAKToolBoxUiInputMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Append prefix</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="SAKCrcAlgorithmComboBox" name="comboBoxAglorithm"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Escape character</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="SAKSpinBox" name="spinBoxEndIndex"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Append suffix</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="SAKCheckBox" name="checkBoxBigEndian">
     <property name="text">
      <string>Big endian CRC</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="SAKAffixesComboBox" name="comboBoxSuffix"/>
   </item>
   <item row="6" column="1">
    <widget class="SAKSpinBox" name="spinBoxStartIndex"/>
   </item>
   <item row="0" column="1">
    <widget class="SAKAffixesComboBox" name="comboBoxPrefix"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>CRC end index</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>CRC algorithm</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="SAKCheckBox" name="checkBoxAppendCrc">
     <property name="text">
      <string>Append CRC</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>CRC start index</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="SAKEscapeCharacterComboBox" name="comboBoxEscape"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SAKCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">SAKCheckBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">SAKSpinBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKEscapeCharacterComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKEscapeCharacterComboBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKAffixesComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKAffixesComboBox.hh</header>
  </customwidget>
  <customwidget>
   <class>SAKCrcAlgorithmComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SAKCrcAlgorithmComboBox.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
